great_expectations/validator/validator.py

```diff
@@ -1248,13 +1248,32 @@ def _catch_exceptions_in_failing_expectation_validations(
 
         return evrs
 
+    @public_api
     def remove_expectation(
         self,
         expectation_configuration: ExpectationConfiguration,
         match_type: str = "domain",
         remove_multiple_matches: bool = False,
         ge_cloud_id: Optional[str] = None,
     ) -> List[ExpectationConfiguration]:
+        """Remove an ExpectationConfiguration from the ExpectationSuite associated with the Validator.
+
+        Args:
+            expectation_configuration: A potentially incomplete (partial) Expectation Configuration to match against.
+            match_type: This determines what kwargs to use when matching. Options are 'domain' to match based
+                on the data evaluated by that expectation, 'success' to match based on all configuration parameters
+                that influence whether an expectation succeeds based on a given batch of data, and 'runtime' to match
+                based on all configuration parameters.
```

Comment:
Could we format this to be in a list? That would make it more readable.

Reply:
done!